src/library/styling/globalStyles.js

```diff
@@ -1,11 +1,10 @@
 import { createGlobalStyle, css } from 'styled-components'
-import raw from 'raw.macro'
 import theme from '../../theme'
 import { hoverState } from './mediaQueries'
 import '@fontsource/open-sans'
 import '@fontsource/open-sans/700.css'
 
-const toastifyCss = raw('react-toastify/dist/ReactToastify.css')
+import toastifyCss from 'react-toastify/dist/ReactToastify.css?raw'
 
```

Comment:
raw.macro was creating issues with Vite, which is fine becasue we can use `?raw` oout of the box to do the same thing